task_list_app/task_list_app/database/Dockerfile

```diff
@@ -5,5 +5,12 @@ ENV POSTGRES_DB task_data
 ENV POSTGRES_USER task_user
 ENV POSTGRES_PASSWORD task_pass
 
+# Specify the location for the PostgreSQL data in the container
+ENV PGDATA /var/lib/postgresql/data
```

Comment:
This is really cool this environment variable is provided.

Based on what I see on line 9 and line 14, I think there is some `-v path:path` flag that needs to be added to the `docker run` command. That flag seems to be needed, but might not be known to someone coming into the project (like me haha). Can you please either:

1. Write a markdown section in the `README` that has the setup instructions
2. Create a script that has the `docker run ...` command in it

Reply:
Done! Added a detailed instruction in the `README`